### DISCUSSION TITLE
Anker Solix E1600 - HM 1500 - OpenDTU on Battery Power Limiter

### DISCUSSION THREAD

**Discussion by markusruebe** [hoylabs/OpenDTU-OnBattery#559] [Q&A]
Hallo, ich nutzen einen HM 1500 mit 4 Modulen. Das Setup wurde um eine Anker E1600 Batterie erweitert. 
Der HM 1500 hat 4 DC Eingänge. an 1 und 2 hängt jeweils ein Paner. An 3 und 4 hängt die Anker Batterie. Leider hat die E1600 weder MQTT noch andere Schnittstellen (ausser der Anker App). Einen Shelly EM3 Pro habe ich auch.
Der Shelly ist erfolgreich in die openDTU eingebunden. Die Batterie kann ich nicht einbinden. Wie kann ich trotzdem den Limiter steuern, um an eine Nulleinspeisung zu kommen?

**Comment by Manos1966** [comment]
Die OpenDTUonBattery ist so ausgelegt, dass alle Hoymiles-Eingaenge auf die gleiche Energie-Quelle angeschlossen sind.

> Wie kann ich trotzdem den Limiter steuern, um an eine Nulleinspeisung zu kommen?

Antwort1: Gar nicht 
Alternative Antwort2: Sehr sehr schwierig und ungenau

OpenDTU steuert den HM-1500 und befiehl ihn eine bestimmte Leistung zu erbringen, z.B. 800W
Diese 800W werden innerhalb der HM1500 auf die zwei MPPTs (1&2 und 3&4) gleichmäßig verteilt, also 400W pro Seite.
Da deine zwei HM-1500 MPTTs auf unterschiedliche Quellen angeschlossen sind, wird es grosse Unterschiede und dadurch grosse Schwankungen geben. Es ist unwahrscheinlich dass es zu einer Nulleinspeisung kommen wird.
Nacht wird sowieso nur eine Seite Strom versorgen, der HM1500 wird nur die Haelfte der notwendige Leistung liefern koennen.
Also, Nulleinspeisung unwahrscheinlich.
Aber Schlimm ist es nicht. Du hast eine Solix E1600 (1,6kWh?) Die Batterie Kapazitaet ist so gering, dass sie fuer die ganze Nacht nicht ausreicht. 
Daher ob deine Solix:
100% Prozent der notwendigen Energie fuer eine Nulleinspeisung fuer ein paar Stunden liefert, 
oder nur 50% Prozent t der notwendigen Energie fuer eine Nulleinspeisung fuer doppelt so viele Stunden, 
liefert, ist es meiner Meinung nach, egal.

**Reply by spcqike** [reply]
> Nacht wird sowieso nur eine Seite Strom versorgen, der HM1500 wird nur die Haelfte der notwendige Leistung liefern koennen.

Ich denke nachts würde es gehen. Solange der Verbrauch < 750W ist. 

OpenDTU prüft ja, wie viele Eingänge grade produzieren (>2W liefern) und skaliert entsprechend. 

Tagsüber wäre es nicht ideal. Außer die Paneele liegen direkt in der Sonne und können den halben Bedarf decken. 


Sauber ist es natürlich nicht. Es würde gelegentlich zu größeren Abweichungen (nach unten) kommen.

**Comment by Manos1966** [comment]
> OpenDTU prüft ja, wie viele Eingänge grade produzieren (>2W liefern) und skaliert entsprechend

Hi @spcqike Weiss du ab welche Version sie das macht? 🎉 Das wäre Super! 👍 (das muss irgendwo in die Wiki notiert werden)
Wenn (Falls) ich mich gut erinnere, die Hoymiles Leistung wird auf MPTT-Basis geteilt, die Anzahl Eingänge-pro-MPTT ist aber unterschiedlich:
HM-Serie: 1 Eingang (HM600) bis 2 Eingänge pro MPTT
HMT-Serie: 2 Eingänge pro MPTT
HMS-Serie: 1 Eingang pro MPTT

**Reply by spcqike** [reply]
Wahrscheinlich seit dem commit 

https://github.com/helgeerbe/OpenDTU-OnBattery/commit/a1da3f98420bd7e4fc5bae818a9226cc25c0a606

**Comment by Manos1966** [comment]
Hallo @markusruebe kannst du ein paar Bilder posten, wir dein System sich verhält?
